docs/sources/alerting/alerting-rules/create-alerts-panels.md

```diff
@@ -0,0 +1,37 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/alerting-rules/create-alerts-panels/
+description: Create alert rules from panels.  Reuse the queries in the panel and create alert rules based on them.
+keywords:
+  - grafana
+  - alerting
+  - panels
+  - create
+  - grafana-managed
+  - data source-managed
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Create alert rules from panels
+weight: 400
+---
+
+## Create alert rules from panels
+
+Create alert rules from any panel type. By doing so, you can reuse the queries in the panel and create alert rules based on them.
```

Comment:
actually, the only panels that support alerts ATM are the time-series and alert list panels 😶

Reply:
so say Create alert rules from any time series panel then?

Reply:
I also see your note and play - that will also need to come out into the new doc, right?